tianshou/env/venvs.py

```diff
@@ -218,10 +222,10 @@ def seed(self,
             which a reproducer pass to "seed".
         """
         self._assert_is_not_closed()
-        if np.isscalar(seed):
-            seed = [seed + _ for _ in range(self.env_num)]
-        elif seed is None:
+        if seed is None:
             seed = [seed] * self.env_num
+        elif not isinstance(seed, list):
+            seed = [seed + _ for _ in range(self.env_num)]
```

Comment:
`_` is not supposed to be used as an actual variable by convention. Please use something else.